examples/docker/pyproject.toml

```diff
@@ -33,23 +33,22 @@ pytest = "*"
 # Used for installing into production, as a post-process step
 # we would like make better integration for this in the future.
 [tool.pixi.feature.prod.host-dependencies]
-pip = "*"
+uv = "0.4.2"
```

Comment:
note: `uv` version 0.4.3 was released today on pypi, but not yet available in conda-forge.

Reply:
Updated in ef83c861a713b7fd9d611f2a4a24fcc34c6189b5